### PR TITLE
fix for #2738

### PR DESCRIPTION
**Pull request by ThamiMemel**
Please test it, i tried to setup a full development environment but i couldn't so i grabbed the react component and test it, it worked for me.
**Related issue**: #2738

### CODE REVIEW

File: app/webpack/shared/components/text_editor.jsx [hunk 1]
```diff
@@ -54,9 +54,7 @@ class TextEditor extends React.Component {
     let textareaOnChange;
     if ( maxLength ) {
       textareaOnChange = e => {
-        if ( e.target.value.length > showCharsRemainingAt ) {
```
Comment (kueda) [inline app/webpack/shared/components/text_editor.jsx:57]: This is just going to remove the `showCharsRemainingAt` functionality and always show the indicator. We need to keep that functionality while also allowing select all -> delete to remove the indicator.

**Comment by ThamiMemel** [comment]
i suspected it would do that because i couldn't use all the component i would like to setup the full dev env but i get the following error **Error: PostgreSQL needs to be installed before setup can be completed.**
i am on Ubuntu 20.04 i have all dependencies installed, also i am using  postgresql-12

**Comment by ThamiMemel** [comment]
thank you for the help,
i'll install ubuntu xanial VM tomorrow and use the .travis.yml as a source.

**Comment by ThamiMemel** [comment]
Hello @kueda 
i managed to setup the databases manually using **bin/setup** instructions then i did the rest of the setup process everything went smoothly .
However when i ran the local server the homepage loaded but when i enter most pages (eg: http://localhost:3000/signup) i get the following error.
**Elasticsearch::Transport::Transport::Errors::NotFound at /signup**

`[404] {"error":{"root_cause":[{"type":"index_not_found_exception","reason":"no such index [development_observations]","resource.type":"index_or_alias","resource.id":"development_observations","index_uuid":"_na_","index":"development_observations"}],"type":"index_not_found_exception","reason":"no such index [development_observations]","resource.type":"index_or_alias","resource.id":"development_observations","index_uuid":"_na_","index":"development_observations"},"status":404}

`
image

- **OS** : Ubuntu 16.04
- **Ruby** : 2.6.5

**Comment by kueda** [comment]
Looks like elasticsearch didn't get set up correctly. Did you go through the steps at https://github.com/inaturalist/inaturalist/wiki/Development-Setup-Guide#elasticsearch? Since you're getting that error I'm guessing you have ES up and running but you just didn't run `rake es:rebuild`.

**Comment by kueda** [comment]
@ThamiMemel, not sure if you're still planning to work on this, but we just changed the default branch from `master` to `main`, so you might need to update this PR or just open another one.

**Comment by kueda** [comment]
@ThamiMemel it's been a month so I'm guessing you're not planning on working on this anymore. I'm going to close this for now and open up the issue, but if you change your mind and want to work on this again, feel free to open a new PR.

**Pull request closed without merging**